### PR TITLE
BotService: update tests

### PR DESCRIPTION
**Pull request by carlosscastro**

### CODE REVIEW

File: src/SDKs/BotService/Management.BotService/Microsoft.Azure.Management.BotService.csproj [hunk 1]
```diff
@@ -6,7 +6,7 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Management.BotService</PackageId>
     <Description>Microsoft Azure Management Bot Service Library</Description>
-    <VersionPrefix>0.9.1-preview</VersionPrefix>
+    <VersionPrefix>0.9.2-preview</VersionPrefix>
     <AssemblyName>Microsoft.Azure.Management.BotService</AssemblyName>
     <PackageTags>Microsoft Azure Bot Service management;Bot;Bot Service;REST HTTP client;windowsazureofficial;netcore451511</PackageTags>
     <PackageReleaseNotes>
```
Comment (dsgouda) [inline src/SDKs/BotService/Management.BotService/Microsoft.Azure.Management.BotService.csproj]: @carlosscastro please update `PackageReleaseNotes`

**Review by dsgouda — CHANGES_REQUESTED** [review]
Looks good apart from a minor comment

File: src/SDKs/BotService/Management.BotService/Customizations/MsaAuthenticator.cs [hunk 2]
```diff
@@ -32,8 +32,7 @@ public MsaAuthenticator(string tenantId)
             this.tenantId = tenantId;
         }
 
-#endif
-#if NETSTANDARD1_4
```
[the remark this answers is not in the document]
Comment (carlosscastro) [reply src/SDKs/BotService/Management.BotService/Customizations/MsaAuthenticator.cs]: @dsgouda

[the remark this answers is not in the document]
Comment (dsgouda) [reply src/SDKs/BotService/Management.BotService/Customizations/MsaAuthenticator.cs]: FullNetFcx corresponds to net framework AFAIK
@shahabhijeet could you confirm?

**Comment by shahabhijeet** [comment]
@carlosscastro in order to run tests as Full Desktop, then you will have to create a new test project that only tragets full desktop like below
https://github.com/Azure/azure-sdk-for-net/tree/master/src/SDKs/Subscription/Subscription.Tests

**Comment by carlosscastro** [comment]
Thanks I'll do that!

**Comment by dsgouda** [comment]
@carlosscastro gentle ping

**Comment by dsgouda** [comment]
@carlosscastro another ping

**Comment by dsgouda** [comment]
@carlosscastro Please reopen the PR when ready. This is required by external customers

**Pull request closed without merging**